Sources/Controllers/MessagesViewController+DataSource.swift

```diff
@@ -53,21 +53,28 @@ extension MessagesViewController: UICollectionViewDataSource {
             fatalError(MessageKitError.nilMessagesDataSource)
         }
 
+        let isMessageCell = indexPath.row == 0
+
         let message = messagesDataSource.messageForItem(at: indexPath, in: messagesCollectionView)
 
-        switch message.data {
-        case .text, .attributedText, .emoji:
-            let cell = messagesCollectionView.dequeueReusableCell(TextMessageCell.self, for: indexPath)
-            cell.configure(with: message, at: indexPath, and: messagesCollectionView)
-            return cell
-        case .photo, .video:
-            let cell = messagesCollectionView.dequeueReusableCell(MediaMessageCell.self, for: indexPath)
-            cell.configure(with: message, at: indexPath, and: messagesCollectionView)
-            return cell
-        case .location:
-            let cell = messagesCollectionView.dequeueReusableCell(LocationMessageCell.self, for: indexPath)
-            cell.configure(with: message, at: indexPath, and: messagesCollectionView)
-            return cell
+        if isMessageCell {
```

Comment:
What's the purpose of here?

Reply:
Thanks @zhongwuzw! This should not be here. I was testing adding more cells and I polluted this PR.